markets/perps-market/contracts/storage/GlobalPerpsMarketConfiguration.sol

```diff
@@ -56,6 +56,11 @@ library GlobalPerpsMarketConfiguration {
          * @notice If set to a larger number (larger than number of collaterals enabled) it means is unlimited.
          */
         uint128 maxCollateralsPerAccount;
+        /**
+         * @dev each collateral used has a strict staleness tolerance that is used when dealing with liquidations since we want the most recent price.
+         * @dev this mapping has the configured tolerance for each collateral.
+         */
+        mapping(uint128 => uint) collateralStalenessTolerances;
```

Comment:
not used. It can be removed

Reply:
and the setter/getter + roll back the setConfiguration to setMax (since is the only configuration)